drgn_tools/locking.py

```diff
@@ -5,9 +5,69 @@
 """
 from typing import Iterable
 
+import drgn
 from drgn import Object
 from drgn import Program
 from drgn.helpers.linux.list import list_for_each_entry
+from drgn.helpers.linux.sched import task_state_to_char
+
+from drgn_tools.bt import bt
+from drgn_tools.task import task_lastrun2now
+
+MUTEX_FLAGS = 0x7
+
+
+def mtx_owner(prog: Program, mtx: drgn.Object) -> drgn.Object:
```

Comment:
I'd argue that the uses of `mtx` here should be `mutex` - again, it's only saving two characters, it's a public API, and using `mutex` feels much clearer.

Reply:
Doen

Reply:
Done